### DISCUSSION TITLE
Is FBW A320 affecting sim rate of MSFS?

### DISCUSSION THREAD

**Discussion by argothus** [flybywiresim/aircraft#8161] [Q&A] [answered]
Hi there,
I mostly fly offline and to save time I'm using the sim rate of msfs.
Recently I noticed, that as i go from x2 to x4 sim rate, after a few moments the sim rate gets automatically decreased to x2.
Has this something to do with the Airbus A320 from FBW?

Greetings

**Comment by hotshotp** [comment, marked as answer]
your fps has to be at least 17 times the simrate to maintain the simrate selected. So 34 fps for 2x, 68 for 4x, etc. This is done because of the way the autopilot works with time passing in MSFS.
